### PR TITLE
feat(map-deck): Add showBuildings to Common.ts

### PR DESCRIPTION
**Pull request by jbrundage**
Signed-off-by: Jaman Brundage <jbrundage372@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [x] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [x] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

## Testing:
This is an "at-runtime" feature. It seems that mapgl doesn't support the ability to add/remove layers after initialization?

### CODE REVIEW

File: packages/map-deck/src/Common.ts [hunk 1]
```diff
@@ -110,6 +111,41 @@ export class Common extends HTMLWidget {
...
+                        ],
+                        "fill-extrusion-base": [
+                            "interpolate", ["linear"], ["zoom"],
+                            15, 0,
+                            15.05, ["get", "min_height"]
+                        ],
+                        "fill-extrusion-opacity": .6
+                    }
+                }, labelLayerId);
+            }
```
Comment (GordonSmith) [inline packages/map-deck/src/Common.ts]: Code won't work if user disables / enables at runtime?

**Comment by GordonSmith** [comment]
@jbrundage bump

File: packages/map-deck/src/Common.ts (hunk 1 above)
Comment (jbrundage) [reply packages/map-deck/src/Common.ts]: Right, once the widget has rendered, this feature can't be toggled. Should we avoid features like this? Or recreate the widget when this feature is toggled?

Comment (GordonSmith) [reply packages/map-deck/src/Common.ts]: Recreate should be fine, its really only a design or user interaction type of thing.

**Comment by jbrundage** [comment]
@GordonSmith it now supports toggling without having to re-render.

**Comment by jbrundage** [comment]
5340C696-82A1-418B-92ED-C85E2450E31F

**Pull request merged**